bedrock/pocket/templates/pocket/contact-info.html

```diff
@@ -15,35 +15,35 @@
+    <h2 class="contact-info-subheader">{{ ftl('contact-api') }}</h2>
+    <p class="contact-info-body-text">{{ ftl('contact-regarding-api', api_email_link='mailto:api@getpocket.com', api_email='api@getpocket.com') }}</p>
 
-    <h2 class="contact-info-subheader">Jobs</h2>
-    <p class="contact-info-body-text">To view all open positions at Pocket, please visit our <a href="https://getpocket.com/jobs">Jobs Page</a>.</p>
+    <h2 class="contact-info-subheader">{{ ftl('contact-jobs') }}</h2>
+    <p class="contact-info-body-text">{{ ftl('contact-view-job-page', jobs=url('pocket.jobs')) }}</p>
 
-    <h2 class="contact-info-subheader">Security</h2>
-    <p class="contact-info-body-text">If you believe you have discovered a security vulnerability in Pocket, please follow Mozilla’s bug reporting process documented on <a href="https://www.mozilla.org/en-US/about/governance/policies/security-group/bugs/">Mozilla’s Security page</a>.</p>
-    <p class="contact-info-body-text">For questions related to security, please contact us at <a href="mailto:security@getpocket.com">security@getpocket.com</a>.</p>
+    <h2 class="contact-info-subheader">{{ ftl('contact-security') }}</h2>
+    <p class="contact-info-body-text">{{ ftl('contact-report-security-vulnerability', security_bug='https://www.mozilla.org/en-US/about/governance/policies/security-group/bugs/') }}</p>
+    <p class="contact-info-body-text">{{ ftl('contact-related-to-security', security_email_link='mailto:security@getpocket.com', security_email='security@getpocket.com') }}</p>
 
-    <h2 class="contact-info-subheader">Follow Us</h2>
-    <p class="contact-info-body-text"><a href="https://blog.getpocket.com/">Pocket Blog</a></p>
-    <p class="contact-info-body-text">Twitter (<a href="https://twitter.com/pocket">@pocket</a>, <a href="https://twitter.com/pocketsupport">@pocketsupport</a>)</p>
-    <p class="contact-info-body-text"><a href="https://www.facebook.com/getpocket">Facebook</a></p>
-    <p class="contact-info-body-text"><a href="https://www.instagram.com/pocket/">Instagram</a></p>
+    <h2 class="contact-info-subheader">{{ ftl('contact-follow-us') }}</h2>
+    <p class="contact-info-body-text"><a href="https://blog.getpocket.com/">{{ ftl('contact-blog') }}</a></p>
+    <p class="contact-info-body-text">{{ ftl('contact-twitter', twitter_main='@pocket', twitter_main_link='https://twitter.com/pocket', twitter_support='@pocketsupport', twitter_support_link='https://twitter.com/pocketsupport') }}</p>
+    <p class="contact-info-body-text"><a href="https://www.facebook.com/getpocket">{{ ftl('contact-facebook') }}</a></p>
+    <p class="contact-info-body-text"><a href="https://www.instagram.com/pocket/">{{ ftl('contact-instagram') }}</a></p>
 
-    <h2 class="contact-info-subheader">Pocket HQ</h2>
+    <h2 class="contact-info-subheader">{{ ftl('contact-hq') }}</h2>
     <address class="contact-info-body-text address">
```

Comment:
Are addresses translatable?

Reply:
@craigcook any idea on this question? I wasn't sure if parts of addresses were (such as the numbers and words like avenue, street, bvld, etc.)

Reply:
Just chatted with Craig - lets leave this as is.